### DISCUSSION TITLE
Expose lock entity attributes as state attributes

### DISCUSSION THREAD

**Discussion by jbouwh** [home-assistant/architecture#860] [Entity Models]
The lock entity has attributes `_attr_is_locked`, `_attr_is_locking`, `_attr_is_unlocked`, `_attr_is_jammed` to determine the state.
These entity attributes are not exposed as state attributes which makes it hard to see in what state a lock is when it is in `jammed` state as only the state is exposed. If we expose the the entity attributes it becomes clear of the lock is in a `locked` stated and in what transition it is when the lock motor gets jammed.

**Comment by thecode** [comment]
If I understand it correctly our suggestion is similar to how we handle cover device, `closed`, `open`, `is closing`, ..., if this is the case I think it would be good to expose these states.
